operator/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java

```diff
@@ -6,6 +6,8 @@
 /** Kubernetes constants. */
 public interface KubernetesConstants {
   String DEFAULT_IMAGE = "container-registry.oracle.com/middleware/weblogic:12.2.1.4";
+  String DEFAULT_EXPORTER_IMAGE = "oracle/weblogic-monitoring-exporter:2.0";
```

Comment:
We can no longer publish images to Docker Hub. Therefore, we've chosen to preference defaults from ghcr.io rather than OCR. Let's use `ghcr.io/oracle/weblogic-monitoring-exporter:2.0`.

Reply:
Done.